workos/event_objects/directory_group.py

```diff
@@ -0,0 +1,46 @@
+from workos.utils.types import JsonDict
+
+
+class DirectoryGroupEvent:
+    def __init__(self, attributes) -> None:
+        self.id: str = attributes["id"]
+        self.name: str = attributes["name"]
+        self.idp_id: str = attributes["idp_id"]
+        self.directory_id: str = attributes["directory_id"]
+        self.organization_id: str = attributes["organization_id"]
+        self.created_at: str = attributes["created_at"]
+        self.updated_at: str = attributes["updated_at"]
+        self.raw_attributes: JsonDict = attributes.get("raw_attributes", {})
```

Comment:
@PavanKulkarni didn't we stop emitting `raw_attributes` for directory group events?

Reply:
Let me confirm this in LD.

Reply:
yes only 3 teams still depend on the [flag](https://app.launchdarkly.com/projects/default/flags/exclude-group-raw-attributes-outgoing/targeting?env=development&env=test&env=production&selected-env=production). That being said we still send `raw_attributes` IIRC, it's just empty.